packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -259,6 +295,34 @@ export class Cluster extends Resource implements ICluster {
     }
   }
 
+  /**
+   * Add default capacity provider strategy for this cluster.
+   *
+   * @param defaultCapacityProviderStrategy cluster default capacity provider strategy. This takes the form of a list of CapacityProviderStrategy objects.
+   *
+   * For example
+   * [
+   *   {
+   *     capacityProvider: 'FARGATE',
+   *     base: 10,
+   *     weight: 50
+   *   }
+   * ]
+   */
+  public addDefaultCapacityProviderStrategy(defaultCapacityProviderStrategy: CapacityProviderStrategy[]) {
+    defaultCapacityProviderStrategy.forEach(dcp => {
+      if (!this._capacityProviderNames.includes(dcp.capacityProvider)) {
+        throw new Error(`Capacity provider with default strategy ${dcp.capacityProvider} is not present in cluster's capacity providers.`);
```

Comment:
This error message doesn't feel super clear to me. The terms used are too close and it feels a little overloaded.